Gemfile.lock

```diff
@@ -191,10 +191,11 @@ GEM
       pry (>= 0.13, < 0.15)
     public_suffix (5.0.1)
     racc (1.6.2)
-    rack (2.2.6.4)
+    rack (3.0.7)
```

Comment:
Are there any concerns about moving major version of rack? It seems like we are still on v2 most places?

Reply:
There is no web ui here, so not concerned.